### ISSUE TITLE
Potentially undesired issues with t-shirts?

### ISSUE THREAD

**Issue by Meandering-Breviger** [CleverRaven/Cataclysm-DDA#56272]
### Describe the bug

See [this video for clarity](https://www.reddit.com/user/Meandering_Breviger/comments/tj1llr/video_for_github_issue/)  (on reddit since it's easier).
If you watch the video you will notice the obvious, the game can't decide on what t-shirt to use and switches between the new variants added in #56041 upon every arrow key press.
Perhaps #56041 can be moved to `tshirt_text` or have `tshirt_text` be merged into `tshirt` as its kinda redundant by this point?

### Steps to reproduce

- Start up character creator
- Go to professions
- Look for something with a t-shirt like Science Club Member 
- Hit the left and right arrow keys.

### Expected behavior

Kinda expected nothing would change and it just be a generic t-shirt.

### Versions and configuration

- Linux Tiles
- Experimental as of 3/20/22

**Comment by NetSysFire** [comment]
Ping @Maleclypse

**Comment by Maleclypse** [comment]
It might be possible to fix it by changing the character creation menu to use a specific t-shirt variant.   I believe there is still a genericized variant t-shirt that could be used for this.

**Comment by anothersimulacrum** [comment]
We probably need a C++ solution to this in the future (only select variants once).

**Comment by ZhilkinSerg** [comment]
UI should probably allow to select item variants, but selecting random variant (as it does now) is also fine if it would be selected only once.
